internal/scheduler/database/migrations/001_initialize_schema.up.sql

```diff
@@ -29,7 +33,7 @@ ALTER TABLE jobs ALTER COLUMN submit_message SET STORAGE EXTERNAL;
 
 CREATE TABLE runs (
     run_id UUID PRIMARY KEY,
-    job_id UUID NOT NULL,
+    job_id TEXT NOT NULL,
```

Comment:
Consistent column type casing. We jobs uses lowercase, runs uses upper

Reply:
done- I've made everything lower case for now